groups/instance/variables.tf

```diff
@@ -49,3 +49,81 @@ variable "db_engine" {
   default     = "postgres"
   type        = string
 }
+
+variable "ebs_root_delete_on_termination" {
+  description = "Whether the volume should be destroyed on instance termination. Defaulted to false"
+  default     = "false"
+  type        = string
+}
+
+variable "ebs_root_encrypted" {
+  description = "Enables EBS encryption on the volume."
+  default     = "true"
+  type        = string
+}
+
+variable "ebs_root_iops" {
+  description = "Amount of provisioned IOPS."
+  default     = 3000
+  type        = number
+}
+
+variable "ebs_root_throughput" {
+  description = "Throughput to provision for a volume in mebibytes per second (MiB/s)"
+  default     = 125
+  type        = number
+}
+
+variable "ebs_root_volume_size" {
+  description = "Size of the volume in gibibytes (GiB)"
+  default     = 20
+  type        = number
+}
+
+variable "ebs_root_volume_type" {
+  description = "Type of volume"
+  default     = "gp3"
+  type        = string
+}
+
+variable "efs_permit_client_root_access" {
+  description = "Enable clients to perform root operations on the filesystem"
+  default     = "true"
+  type        = string
+}
```

Comment:
This variable needs to be a `bool` type.

Reply:
Completed as required. Removed this variable and argument from the instance resource. Converted the efs_permit_client_root_access variable to a bool.